src/components/TopBar.vue

```diff
@@ -61,10 +60,42 @@ const formatWalletBalance = (): String => {
           <span class="default-text"> Ethereum </span>
           <img alt="Chevron Down" src="@/assets/chevronDown.svg" />
```

Comment:
This image will be dynamic, it can be a white chevron down or a black chevron up. If the menu is selected you need to show the black chevron up. Please fix this, according to design above:
![image](https://user-images.githubusercontent.com/74625814/208738668-573c5b20-07d8-44e5-b30f-5168a0a418f0.png)